### ISSUE TITLE
confused about where to push changes

### ISSUE THREAD

**Issue by williamh** [gentoo/gentoo-rust#92]
I'm confused now about where to push changes or where to pull from:

Heather/gentoo-rust is still the base repository, so should I still pull from there, and what about where to push changes?

**Comment by williamh** [comment]
Also, I think the rust repository under the Gentoo organization should be named Gentoo-rust since it is a fork of this repository.

**Comment by Heather** [comment]
@williamh I'm sorry for late reply but no one replied on migration thread previously and I'm not sure if everyone is ready for migration to `Gentoo` org. I don't think repository under `Gentoo` should be renamed. Currently we're still being in migration stage, also me or someone else should modify readme about this all. This repository is main (for now) and another repository is in sync...

Due migration stage you should commit to `Gentoo/rust` repository but before it you should ensure that it's up-to-day synchronized with this repository. If you don't care about sync process then don't commit to `Gentoo` repository.

**Comment by williamh** [comment]
The reason I think the repository under Gentoo should be gentoo-rust, rust-overlay or something similar is because just calling it rust confuses it with upstream's rust-lang/rust repository. It is inconvenient to be forced to rename a repository locally or store them in separate directories. Please reconsider the name of the Gentoo/rust repository.

Thanks.

**Comment by Heather** [comment]
@williamh I see but there are Gentoo/kde , qt, gnome, etc...

**Comment by williamh** [comment]
Sure, but I think the difference is that there are not individual packages called gnome, kde, dotnet, etc, where we have a package called rust.

**Comment by Heather** [comment]
@williamh I see but still... I need more opinions about it to rename it there

**Comment by jauhien** [comment]
My two cents for renaming. Rust-overlay is ok.

**Comment by jauhien** [comment]
Bump. What about renaming https://github.com/gentoo/rust to gentoo-rust and setting it as the official repo in layman? @Heather: please, rename it (if you have no complains, or we can discuss it more here) and then I'll proceed with appropriate layman bug.

**Comment by Heather** [comment]
@jauhien well gentoo-rust is fine however I still prefer just rust, you can do it and also you can change layman target to gentoo organization - just reply in this issue when it will be complete and then update readme and sync this repository for future contributions

**Comment by jauhien** [comment]
Renamed, bug for layman list: https://bugs.gentoo.org/show_bug.cgi?id=559702

**Comment by jauhien** [comment]
Done. Also I've migrate some issues. I didn't find how to mark new repo as main (not fork). I'll update README later and try to see what else we need. Also would be good to clock possibility to create new issues and PR in this repo.

**Comment by Heather** [comment]
@jauhien thank you, marking repository as "main" on github is not necessary but it will be good to write it down in readme file. not sure but maybe I will need to remove this repository and re-fork it from gentoo later to do it but this will clean up issues and pr, for now it will be good to have some "migrating support" in this repository for some time. In addition - later we can migrate to gentoo git server and just keep github one as mirror.

**Comment by jauhien** [comment]
I updated the README.
